src/languages/es.ts

```diff
@@ -1350,6 +1350,11 @@ export default {
         approverInMultipleWorkflows: 'Este miembro ya pertenece a otro flujo de aprobación. Cualquier actualización aquí se reflejará allí también.',
         approverCircularReference: ({name1, name2}: ApprovalWorkflowErrorParams) =>
             `<strong>${name1}</strong> ya aprueba informes a <strong>${name2}</strong>. Por favor, elige un aprobador diferente para evitar un flujo de trabajo circular.`,
+        emptyContent: {
```

Comment:
[Waiting for confirmation](https://expensify.slack.com/archives/C01GTK53T8Q/p1724864006019869)

Reply:
Approved 👌